openedx/core/djangoapps/auth_exchange/tests/test_views.py

```diff
@@ -150,22 +151,27 @@ def setUp(self):
         self.user = UserFactory()
         self.oauth2_client = Application.objects.create(client_type=Application.CLIENT_CONFIDENTIAL)
 
-    def _verify_response(self, access_token, expected_status_code, expected_cookie_name=None):
+    def _verify_response(self, access_token, expected_status_code,
+                         token_type='Bearer', expected_cookie_name=None, data=None):
```

Comment:
Nit: if this argument order makes the most sense to you, having callers use this consistent order would be nice. Lots of callers use `data` before `token_type`, as an example.